contrib/golang/filters/http/source/cgo.cc

```diff
@@ -0,0 +1,134 @@
+#include "contrib/golang/filters/http/source/golang_filter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Golang {
+
+//
+// These functions may be invoked in another go thread,
+// which means may introduce race between go thread and envoy thread.
+//
+
+absl::string_view copyGoString(void* str) {
+  if (str == nullptr) {
+    return "";
+  }
+  auto goStr = reinterpret_cast<GoString*>(str);
+  return absl::string_view(goStr->p, goStr->n); // NOLINT(modernize-return-braced-init-list)
+}
+
+extern "C" {
+
+void moeHandlerWrapper(void* r, std::function<void(std::shared_ptr<Filter>&)> f) {
```

Comment:
I'm not sure what `moe` means. I would replace with something more obvious like `envoy_go_filter_` or something like that. Here and elsewhere. This will make the exports more clear.

Reply:
Done. use `envoy_go_filter_` or `envoyGoFilter` instead of all `moe`